I-D-Profile-Negotiation.xml

```diff
@@ -282,94 +282,110 @@ Preference-Applied: profile=&lt;urn:example:schema&gt;
-					of XML Schema 1.1-1</xref>) -- a list of pairs of URI references separated by
-					whitespace similar to the syntax used in the xsi:schemaLocation attribute. If 
-					a client uses the "Accept-Profile" header to specify one or more
-					profiles it is willing to accept and a server does not use the "Profile" 
-					header to specify which profile the returned content conforms to, 
-					the client MAY process the returned content as it deems fit.</t>
-				<t>If a client uses the "Profile" header field to indicate the profile the
-					payload conforms to (e. g. in an HTTP POST or PUT request) and the
-					server cannot process content conforming to that profile, the
-					server SHOULD send a 406 (Not acceptable) response together with an "Accept-Profile"
-					header field (including q-values) to indicate the profiles it 
-					can process. Reasons for not sending a 406 response in such a case might be that
-					the the processing of the message payload leads to an internal server error. If in
-					such a case the server does not implement profile negotiation, the server is
-					more likely to return a 500 (Internal server error) response instead of 406.</t>
-				<t><xref target="abnf_schema"/> describes the syntax (Augmented Backus-Naur Form) of the
+					message conforms to.
+					The profile is identified by a URI reference that MUST be an absolute URI.
+					If a user agent uses the "Accept-Profile" header
+					to specify one or more profiles it is willing to accept
+					and a server does not use the "Content-Profile"	header to
+					specify which profile the returned content conforms to, 
+					the user agent MAY process the returned content as it deems fit.</t>
+				<t>The Content-Profile field MAY contain a comma-separated list of profile-URIs.
+					In that case the recipient SHOULD interpret this as if the payload of the message
+				  conforms to all of the profiles identified by the supplied profile URIs.
+				  Instead of sending a comma-separated list of profile URIs in the Content-Profile header field,
+					and agent MAY choose to send each profile URI in its own Content-Profile header field.
+					Receivers MUST treat those two variants as equal.</t>
+				<t>If a user agent uses the "Content-Profile" header field
+					to indicate the profile the	payload conforms to
+					(e. g. in an HTTP POST or PUT request)
+					and the	server cannot process content conforming to that profile,
+					the	server SHOULD send a 406 (Not acceptable) response
+					together with an "Accept-Profile"	header field (including q-values)
+					to indicate the profiles it can process.
+					Reasons for not sending a 406 response in such a case might be
+					that the the processing of the message payload leads to an internal server error.
+					If in	such a case the server does not implement profile negotiation,
+					the server is	more likely to return a 500 (Internal server error) response instead of 406.</t>
```

Comment:
I don't think that's true. It would just ignore it and not do 500 or 406.

Reply:
Ignore. If the server doesn't support, it will likely doesn't know it doesn't support.

Reply:
I still think that if a server supports profile negotiation and a client tries to POST or PUT content using a profile that the server cannot handle it would be appropriate for the server to answer with a 406 and a list of profiles it can accept.
It would be OK not to talk about servers that don't know anything about profiles.

Reply:
Agreed.